docs-source/content/quickstart/install.md

```diff
@@ -67,15 +48,20 @@ $ helm install traefik-operator traefik/traefik \
 
 3.  Use `helm` to install and start the operator from the directory you just cloned:	 
 
-
     ```bash
     $ helm install sample-weblogic-operator kubernetes/charts/weblogic-operator \
       --namespace sample-weblogic-operator-ns \
-      --set image=oracle/weblogic-kubernetes-operator:3.0.0 \
+      --set image=oracle/weblogic-kubernetes-operator:3.1.0 \
       --set serviceAccount=sample-weblogic-operator-sa \
-      --set "domainNamespaces={}" \
+      --set "enableClusterRoleBinding=true" \
+      --set "domainNamespaceSelectionStrategy=LabelSelector" \
+      --set "domainNamespaceLabelSelector=weblogic-operator\=enabled" \
       --wait
     ```
+    
+    This Helm release deploys the operator and configures it to manage Domains in any Kubernetes namespace with the label, "weblogic-operator=enabled". Because of the "enableClusterRoleBinding" option, the operator will have privilege in all Kubernetes namespaces. This simplifies adding and removing managed namespaces as you will only have to adjust labels on those namespaces. If you want to limit the operator's privilege to just the set of namespaces that it will manage then remove this option, but this will mean that the operator only has privilege in the set of namespaces that match the selection strategy at the time the Helm release was installed or upgraded.
```

Comment:
If you want to limit the operator's privilege to just the set of namespaces that it will manage then -> If you want to limit the operator's privilege to just the set of namespaces that it will manage, then (need a comma before then; also helps readability of this long sentence)